samples/Samples.FakeAzureAppServices/Samples.FakeAzureAppServices.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <LoadManagedProfilerFromProfilerDirectory>true</LoadManagedProfilerFromProfilerDirectory>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ProjectReference Include="..\..\src\Datadog.Trace\Datadog.Trace.csproj" />
+  </ItemGroup>
+
+  <ItemGroup>
+    <Reference Include="System.Net.Http" />
+  </ItemGroup>
+
+  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+    <None Include="$(ProjectDir)$(OutputPath)Samples.FakeAzureAppServices.exe"
```

Comment:
What is the purpose of this? It seems redundant since it should be built by the project.

Reply:
Oh I see, you're duplicating the executable but saving it as `Samples.FakeTraceAgent.exe`. Resolving this.